src/backends/tensorflow/tensorflow_backend_tf.cc

```diff
@@ -695,7 +703,7 @@ TRTISTF_ModelCreateFromSavedModel(
   // available in tensorflow/cc/saved_model/loader.cc so we use
   // allocator_type in pass in the gpu_device we want and then
   // loader.cc (our modified version) will use that to
-  // SetDefaultDevice appropriately.
+  // SetDefaultDevice appropriately. 
```

Comment:
You added a space to the end of the line here. Did you run the formatter?

Reply:
I might have missed running formatter after modifying this file. Fixed it.